### DISCUSSION TITLE
Guess 🎥🍿 catchphrase

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#866] [Galaxy Trivia] [answered]
> 🚶ahead, make my 🌞

**Bonus:** Name the character and actor.

**Comment by Carol42** [comment]
"Go ahead, make my day" is a catchphrase from the 1983 film Sudden Impact, spoken by the character Harry Callahan, played by Clint Eastwood.

**Comment by peterbaccaro** [comment, marked as answer]
"**Go ahead, make my day**" is a catchphrase from the 1983 film Sudden Impact, spoken by the character **Harry Callahan**, played by **Clint Eastwood**. 

The iconic line was written by John Milius whose writing contributions to the film were uncredited, but has also been attributed to Charles B. Pierce, who wrote the film's story, and to Joseph Stinson, who wrote the screenplay. 

In 2005, the catchphrase was chosen as No. 6 on the American Film Institute list AFI's 100 Years...100 Movie Quotes.
